dotcom-rendering/README.md

```diff
@@ -92,6 +94,12 @@ If you're new to JavaScript projects, if you're trying to integrate with other a
 
 You can see a _web only_ architecture diagram by running `make arch-diagram`. It will give you an overview of the current server and browser web architecture.
 
+### UI Design System
```

Comment:
Nice one! 🙌 Do you think it's also worth adding source link in this section? https://theguardian.design/2a1e5182b/p/300696-

cc @HarryFischer

Reply:
Yep I think this is useful - as Oliver mentioned it isn't maintained so the hex values are probably out of date now but I think it gives a good overview of our design language. thanks

Reply:
thanks for this! Just to clarify, the overview link (design.theguardian.com) isn't maintained, but I assume that the Source links Ioanna and Oliver shared are maintained, right?

So I might add something like:

```
This resource is not maintained so it should not be taken as authoritative on details, but most of it still applies and it gives a good high-level overview. For more detailed and up-to-date information, there is also the [Guardian Source](https://theguardian.design/2a1e5182b/p/300696-) site and [Source developer guide](https://guardian.github.io/source/?path=/story/source--page).
```

Reply:
What do you think about including the links to the up-to-date docs first?

> [Source](https://theguardian.design) is the Guardian's design system. For detailed and up-to-date information on how to use it, see the [Source developer guide](https://guardian.github.io/source).
>
> For a high-level overview of some of the key ideas behind the design of the Dotcom website, see [design.theguardian.com](https://design.theguardian.com/). This resource is not maintained so it should not be taken as authoritative on details

Reply:
yep that makes a lot of sense! just done that now, thanks! 👍

Reply:
@bryophyta It's really helpful. Thank you very much!

Reply:
glad it was helpful, thanks for asking!